cfn-resources/profile/profile.go

```diff
@@ -30,9 +30,10 @@ const (
 )
 
 type Profile struct {
-	PublicKey  string `json:"PublicKey"`
-	PrivateKey string `json:"PrivateKey"`
-	BaseURL    string `json:"BaseUrl,omitempty"`
+	DebugClient *bool  `json:"DebugClient,omitempty"`
```

Comment:
If we agree on this format, I will update the release profile and test profiles to include the `DebugClient` in their profile secrets.
This way, we will get the debug logs in the s3 bucket when contract tests are run during release & contract testing in CI

Reply:
where is the bucket/url defined?  what happens if a customer wants / can use it?

Reply:
- For contract testing in CI. The logs are `inline`
- For releasing, the logs are in the `mdb-cfn-publish` bucket (configured in [cfn-publishing-helper.config](https://github.com/mongodb/mongodbatlas-cloudformation-resources/blob/CLOUDP-275212_rp_schema_updates/cfn-resources/cfn-publishing-helper.config#L3))
- For customers, they can configure Cloudwatch logging when [activating the type](https://boto3.amazonaws.com/v1/documentation/api/latest/reference/services/cloudformation/client/activate_type.html#) (see `LoggingConfig` ). However, I'm not 100% sure what type of logs these are. I'll try to test this out